src/nvim/globals.h

```diff
@@ -1051,10 +1051,6 @@ EXTERN int typebuf_was_filled INIT(= FALSE);      /* received text from client
 EXTERN int term_is_xterm INIT(= FALSE);         /* xterm-like 'term' */
```

Comment:
It looks like this can be removed too, although I'm not sure what others think about deviating from patches, but I searched Vim's history and this hasn't been touched in a long time.